tofu/spectro/_rockingcurve.py

```diff
@@ -118,9 +123,9 @@ def compute_rockingcurve(
     if use_non_parallelism is None:
         use_non_parallelism = False
     if alpha_limits is None:
-        alpha_limits = np.r_[-(5/60)*np.pi/180, (5/60)*np.pi/180]
+        alpha_limits = np.r_[-(3/60)*np.pi/180, (3/60)*np.pi/180]
     if na is None:
-        na = 51
+        na = 41
     nn = (na/2.)
     if (nn % 2) == 0:
         nn = int(nn - 1)
```

Comment:
Simpler to do it the other way around (use `nn` as input argument, then build `na` so it is always an odd number):

```
if nn is None:
    nn = 20

na = 2*nn + 1
```

If you fear that `nn` is not explicit enough as a argument name, you can rename it differently